apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -126,7 +126,7 @@ export const pageRouter = router({
               title,
               permalink,
               siteId,
-              parentId: String(folderId),
+              ...(folderId && { parentId: String(folderId) }),
```

Comment:
nit: prefer explicit instead of spread:

```suggestion
              folderId: folderId ? String(folderId) : undefined
```

Reply:
but hor why need the string cast in the first place?

Reply:
we migrated to `BigInt` in #365 - this did a wrongful `String` cast. i've went ahead and looked through usages in the codebase and for nullish `stirng | null | undefined`, i went ahead and did this check.